### PR TITLE
Fix mips/baikal template

### PR DESCRIPTION
**Pull request by anton-bondarev**

### CODE REVIEW

File: src/drivers/common/periph_memory_alloc.h [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * @file
+ *
+ * @date Mar 12, 2020
+ * @author Anton Bondarev
+ */
+
+#ifndef SRC_DRIVERS_COMMON_PERIPH_MEMORY_ALLOC_H_
+#define SRC_DRIVERS_COMMON_PERIPH_MEMORY_ALLOC_H_
+
+extern void *periph_memory_alloc(size_t len);
```
Comment (Deryugin) [inline src/drivers/common/periph_memory_alloc.h]: Include stddef.h to make sure size_t is defined, please

File: src/drivers/common/periph_memory_alloc.c [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	}
+
+	return mem;
+}
+#endif /* NOMMU */
+
+void periph_memory_free(void *ptr) {
+	sysfree(ptr);
+}
+
```
Comment (Deryugin) [inline src/drivers/common/periph_memory_alloc.c]: Trailing blank line

File: src/drivers/common/periph_memory_alloc.c [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+#ifndef NOMMU
+
+#include <drivers/common/memory.h>
+#include <hal/mmu.h>
+#include <kernel/task/kernel_task.h>
+#include <kernel/task/resource/mmap.h>
+
+#include <mem/mmap.h>
+
```
Comment (Deryugin) [inline src/drivers/common/periph_memory_alloc.c]: Double blank line

File: src/drivers/common/periph_memory_alloc.c [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	return mem;
+}
+#else
+void *periph_memory_alloc(size_t len) {
+	void *mem = sysmemalign(sizeof(void *), len);
+	if (!mem) {
+		return NULL;
+	}
+
+	return mem;
```
Comment (Deryugin) [inline src/drivers/common/periph_memory_alloc.c]: Can this function be reduced to `return sysmalloc(len)`?

File: src/drivers/common/memory_impl_stub.h [hunk 5]
```diff
@@ -18,4 +18,6 @@ struct periph_memory_desc {
 
 #define PERIPH_MEMORY_DEFINE(_mem_desc, mem_base, mem_len)
 
+extern int periph_desc(struct periph_memory_desc **buff);
```
Comment (Deryugin) [inline src/drivers/common/memory_impl_stub.h]: Looks like this function is not implemented for stub module.

File: src/drivers/common/periph_memory_alloc.h (hunk 1 above)
Comment (anton-bondarev) [reply src/drivers/common/periph_memory_alloc.h]: Thanks, added

File: src/drivers/common/periph_memory_alloc.c (hunk 3 above)
Comment (anton-bondarev) [reply src/drivers/common/periph_memory_alloc.c]: done

File: src/drivers/common/periph_memory_alloc.c (hunk 4 above)
Comment (anton-bondarev) [reply src/drivers/common/periph_memory_alloc.c]: thanks, done as `return sysmemalign(sizeof(void *), len);`

File: src/drivers/common/periph_memory_alloc.c (hunk 2 above)
Comment (anton-bondarev) [reply src/drivers/common/periph_memory_alloc.c]: done

**Pull request merged**